Services/OperationService/Commands/DisableBotCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DTO;
+using DTO.RestRequests;
+using Interfaces;
+using Kernel;
+using Kernel.CustomExceptions;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace OperationService.Commands
+{
+    public class DisableBotCommand : ICommand<DisableBotRequest, bool>
+    {
+        private readonly IRequestClient<DisableBotRequest> client;
+
+        public DisableBotCommand([FromServices] IRequestClient<DisableBotRequest> client)
+        {
+            this.client = client;
+        }
+
+        private async Task<bool> DisableBot(DisableBotRequest request)
+        {
+            var response = await client.GetResponse<OperationResult<bool>>(request);
+
+            return OperationResultHandler.HandleResponse(response.Message);
+        }
+
+        public async Task<bool> Execute(DisableBotRequest request)
+        {
+            var disableBotresult = await DisableBot(request);
```

Comment:
I guess that we should use it in try catch because result wrapper throws exception with needed status and message in case of error, it does not just return false

Reply:
add handler